src/vs/workbench/browser/parts/statusbar/statusbarPart.ts

```diff
@@ -184,11 +151,20 @@ class StatusbarPart extends Part implements IStatusbarEntryContainer {
 		@IWorkbenchLayoutService layoutService: IWorkbenchLayoutService,
 		@IContextMenuService private contextMenuService: IContextMenuService,
 		@IContextKeyService private readonly contextKeyService: IContextKeyService,
-		@IHoverService private readonly hoverService: IHoverService,
-		@IConfigurationService private readonly configurationService: IConfigurationService
 	) {
 		super(id, { hasTitle: false }, themeService, storageService, layoutService);
 
+		this.hoverDelegate = instantiationService.createInstance(WorkbenchHoverDelegate, 'element', true);
+		this.hoverDelegate.setOptions((_, focus?: boolean) => {
```

Comment:
Maybe nicer if this was a callback I can pass in when constructing the hover delegate?